src/test/java/ru/ayaz/VideoTest.java

```diff
@@ -0,0 +1,55 @@
+package ru.ayaz;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public class VideoTest {
+
+    private Video video;
+
+    @Before
+    public final void before() {
+        this.video = new Video("FunnyCats", mock(File.class));
```

Comment:
&#x1F34E; **Issue**
@Before method isn't meant to be used in cases where you have a lot of tests in your test class. And that is because these tests don't fully intersect in their setup.

Hence, you end up with some setup logic in @Before, other setup logic in @Test and its not very convenient to run from one to the other to understand/debug it.

You might argue that if we put @Before into each @Test that requires it we'll end up with code duplication - that is true, but its not that bad, duplication piece is small and it is in tests after all (we'll figure out discrepancies one we run them).

Only when each of your tests can completely relay its initialization logic upon @Before should we use @Before.

Its a matter of opinion mostly, and here is mine )